app/components/questioncard.jsx

```diff
@@ -1,24 +1,32 @@
-import React from 'react';
+import PropTypes from 'prop-types';
 
 function QuestionCard({ question, answers, onAnswer }) {
   return (
-    <div className="card mx-auto w-full max-w-4xl rounded-xl bg-white p-10">
-      <h2 className="question mb-10 text-center text-2xl font-bold text-gray-800">
+    <div className="mx-auto w-full max-w-4xl rounded-xl bg-white p-10">
+      <h2 className="mb-10 text-center text-2xl font-bold text-gray-800">
         {question}
       </h2>
-      <div className="answers-container flex flex-col items-center gap-3">
-        {Object.keys(answers).map((key) => (
-          <button
-            key={key}
-            className="answer w-full max-w-3xl cursor-pointer rounded-lg border border-gray-100 bg-gray-100 p-1 text-lg shadow-md  transition-colors duration-300 focus:outline-none"
-            onClick={() => onAnswer(key)}
-          >
-            {key}: {answers[key]}
-          </button>
-        ))}
+      <div className="flex flex-col items-center gap-3">
+        <div className="flex flex-col items-center gap-3">
+          {Object.keys(answers).map((key) => (
+            <button
+              className="w-full max-w-3xl cursor-pointer rounded-lg border border-gray-100 bg-gray-100 p-1 text-lg shadow-md transition-colors duration-300 focus:outline-none"
+              onClick={() => onAnswer(key)}
+              key={key}
+            >
+              {key}: {answers[key]}
+            </button>
```

Comment:
When I go to the previous question, I can't see which answer I picked. Is this something for this task or different one (and it is created?)